bin/ansible-inventory-grapher

```diff
@@ -105,7 +105,7 @@ def options_parser():
 
 
 def labelescape(name):
-    return name.replace("-", "_").replace(".", "_")
+    return "\"%s\"" % name.replace("-", "_").replace(".", "_")
```

Comment:
The `replace`s were to avoid similar parsing issues. If we quote the labels, do we need the `replace`s any more?

Reply:
`replace` is not needed when using double quotes